lib/ansible/modules/system/known_hosts.py

```diff
@@ -323,20 +325,21 @@ def compute_diff(path, found_line, replace_or_add, state, key):
     diff['after'] = ''.join(lines)
     return diff
 
+
 def main():
 
     module = AnsibleModule(
         argument_spec = dict(
-            name      = dict(required=True,  type='str', aliases=['host']),
-            key       = dict(required=False,  type='str'),
+            name      = dict(required=True, type='str', aliases=['host']),
+            key       = dict(required=False, type='str'),
             path      = dict(default="~/.ssh/known_hosts", type='path'),
-            hash_host = dict(required=False, type='bool' ,default=False),
-            state     = dict(default='present', choices=['absent','present']),
-            ),
-        supports_check_mode = True
-        )
+            hash_host = dict(required=False, type='bool', default=False),
+            state     = dict(default='present', choices=['absent', 'present']),
```

Comment:
It's a matter of aesthetics, but I do think the `argument_spec` is clearly laid out and very readable as it is (i.e. with all the `=` lining up). Which is why I requested it be left alone.